Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -532,9 +532,11 @@ function handleZoom(
     );
   }
 
+  var diff = movement.endPosition.y - movement.startPosition.y;
+
   // distanceMeasure should be the height above the ellipsoid.
   // The zoomRate slows as it approaches the surface and stops minimumZoomDistance above it.
-  var minHeight = object.minimumZoomDistance * percentage;
+  var minHeight = diff < 0 ? 0 : object.minimumZoomDistance * percentage;
```

Comment:
For readability, can you either a) break this out into a descriptive variable, or 2) expand on the comment above?

Reply:
Changed it to:

```js
  // distanceMeasure should be the height above the ellipsoid.
  // When approaching the surface, the zoomRate slows and stops minimumZoomDistance above it.
  var approachingSurface = diff > 0;
  var minHeight = approachingSurface ? object.minimumZoomDistance * percentage : 0;
  var maxHeight = object.maximumZoomDistance;
```